src/main/resources/theme/chat/channel_tab.fxml

```diff
@@ -15,7 +15,8 @@
      fx:controller="com.faforever.client.chat.ChannelTabController">
   <VBox maxHeight="1.7976931348623157E308" maxWidth="1.7976931348623157E308" minHeight="0.0" minWidth="0.0" onKeyReleased="#onKeyReleased">
     <children>
-      <SplitPane dividerPositions="0.8" maxWidth="1.7976931348623157E308" VBox.vgrow="ALWAYS">
+        <SplitPane maxWidth="1.7976931348623157E308" VBox.vgrow="ALWAYS"
+                   fx:id="splitPane"> <!--dividerPositions="0.8" set to fixed px width in code-->
```

Comment:
I don't think comments in FXML are a good idea anyway. Might get lost due to SceneBuilder logic anytime, also you don't see it in SceneBuilder. I talked about the fixed px above already.